test/__snapshots__/mathml-spec.js.snap

```diff
@@ -74,6 +74,41 @@ exports[`A MathML builder should make prime operators into <mo> nodes 1`] = `
 
 `;
 
+exports[`A MathML builder should output \\limsup_{x \\rightarrow \\infty} correctly 1`] = `
+
+<math>
+  <semantics>
+    <mrow>
+      <msub>
```

Comment:
I think this should actually be `munder`.

Reply:
Actually this is right b/c the tests in mathml-spec.js don't use display mode by default.

Reply:
Is it worth saying `in \textstyle` in the `exports` key above to avoid this confusion in the future?